### PR TITLE
Initial commit of Python tool to get a list of those who RSVPed for a meetup

### PR DESCRIPTION
**Pull request by silverdrake11**
Just fill out top part

### CODE REVIEW

File: meetup/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# meetup rsvps
+This tool prints out a list of full names of those who RSVPed yes for the given meetup. The names are those as shown in the Chipy user profile section where the full name is asked.
+
+# requirements
+- Requests
+- Beautiful Soup
```
Comment (alysivji) [inline meetup/README.md:6]: which version? maybe it'll be useful to have a `requirements.txt`

**Review by alysivji — CHANGES_REQUESTED** [review]
Could you add a bit more context for what this PR is doing at a high level (title) and update the description to talk about what you would like to merge in?

File: meetup/README.md (hunk 1 above)
Comment (silverdrake11) [reply meetup/README.md:6]: Any version works it's very basic functionality

**Comment by silverdrake11** [comment]
The changes here are adding the README with instructions on what it does and how to use it, and refactored some of the code. I will update the title

**Review by alysivji — APPROVED** [review]
LGTM

**Pull request merged**